Content.Server/Chat/Systems/AnnounceOnSpawnSystem.cs

```diff
@@ -0,0 +1,22 @@
+using Content.Server.Chat;
+
+namespace Content.Server.Chat.Systems;
+
+public sealed class AnnounceOnSpawnSystem : EntitySystem
+{
+    [Dependency] private readonly ChatSystem _chat = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AnnounceOnSpawnComponent, ComponentInit>(OnInit);
+    }
+
+    private void OnInit(EntityUid uid, AnnounceOnSpawnComponent comp, ComponentInit args)
+    {
+        var message = Loc.GetString(comp.Message);
+        var sender = comp.Sender != null ? Loc.GetString(comp.Sender) : "Central Command";
+        _chat.DispatchGlobalAnnouncement(message, sender, playSound: true, comp.Sound, comp.Color);
+    }
+}
```

Comment:
This should be mapinit so it doesn't announce every time the game is saved and loaded.